.changeset/sweet-chairs-knock.md

```diff
@@ -0,0 +1,12 @@
+---
+"@firebase/firestore": patch
+"@firebase/functions": patch
+"@firebase/storage": patch
+"@firebase/auth-compat": patch
```

Comment:
Can you remove these 5? 3 are exp and 2 are excluded from publishing. Not sure why changeset automatically includes them.
```
Found ignored packages: @firebase/auth-compat @firebase/auth-exp @firebase/functions-exp firebase-messaging-integration-test @firebase/changelog-generator
```